### PR TITLE
[Feature] Add windows CI

### PR DESCRIPTION
**Pull request by ZwwWayne**
Add Windows CI in GitHub actions.

### CODE REVIEW

**Comment by fcakyon** [comment]
@ZwwWayne how is this pr going? is it related with prebuilt binaries for windows?

**Comment by ZwwWayne** [comment]
> @ZwwWayne how is this pr going? is it related with prebuilt binaries for windows?

Yes, we plan to merge it in the next release. The next step is to release prebuilt binaries for windows.

**Comment by ZwwWayne** [comment]
LGTM, invite @innerlee to have a look.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -279,6 +279,36 @@ jobs:
...
+          - torch: 1.7.1
+            torchvision: 0.8.2
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+      # - name: Install system dependencies
+      #   run: brew install ffmpeg jpeg-turbo
```
Comment (hellock) [inline .github/workflows/build.yml:297]: remove useless comments

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -279,6 +279,36 @@ jobs:
...
           fail_ci_if_error: false
 
+  build_windows:
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        torch: [1.7.1]
+        include:
+          - torch: 1.7.1
+            torchvision: 0.8.2
```
Comment (hellock) [inline .github/workflows/build.yml:289]: I suggest also including the latest version.

File: mmcv/runner/hooks/logger/pavi.py [hunk 3]
```diff
@@ -90,7 +89,6 @@ def after_run(self, runner):
         if self.add_last_ckpt:
             ckpt_path = osp.join(runner.work_dir, 'latest.pth')
             if osp.isfile(ckpt_path):
-                ckpt_path = osp.join(runner.work_dir, os.readlink(ckpt_path))
```
Comment (hellock) [inline mmcv/runner/hooks/logger/pavi.py:93]: This line may not be removed directly.

File: .github/workflows/build.yml (hunk 1 above)
Comment (zhouzaida) [reply .github/workflows/build.yml:297]: got it

File: .github/workflows/build.yml (hunk 2 above)
Comment (zhouzaida) [reply .github/workflows/build.yml:289]: done

File: mmcv/runner/hooks/logger/pavi.py (hunk 3 above)
Comment (zhouzaida) [reply mmcv/runner/hooks/logger/pavi.py:93]: fixed

**Review by hellock — APPROVED** [review]

**Pull request merged**